### PR TITLE
MAUI-4127- Included changes in DataForm UG images 

### PR DESCRIPTION
**Pull request by Kerubananthan3761**
## Description

Included renaming changes of DataForm images

### CODE REVIEW

File: MAUI/DataForm/layout.md [hunk 1]
```diff
@@ -19,11 +19,11 @@ By default, the data form arranges the fields one by one. It is applicable for b
 
 When the label position is Left, the linear layout is shown as follows:
 
-![linear-layout-left](images/layout/linear-layout-left.png)
+![Linear layout left in .NET MAUI DataForm.](images/layout/linear-layout-left.png)
```
Comment (karthikraja-arumugam) [inline MAUI/DataForm/layout.md:22]: Linear layout left label position in .NET MAUI DataForm.

File: MAUI/DataForm/layout.md [hunk 2]
```diff
@@ -19,11 +19,11 @@ By default, the data form arranges the fields one by one. It is applicable for b
...
 
 When the label position is Top, the linear layout is shown as follows:
 
-![linear-layout-top](images/layout/linear-layout-top.png)
+![Linear layout top in .NET MAUI DataForm.](images/layout/linear-layout-top.png)
```
Comment (karthikraja-arumugam) [inline MAUI/DataForm/layout.md:26]: Linear layout top label position in .NET MAUI DataForm.

File: MAUI/DataForm/localization.md [hunk 3]
```diff
@@ -44,22 +44,22 @@ Follow these steps to localize the `DataForm` based on the [CurrentUICulture](ht
...
 
 5. Now, click `Add` to add the resource file in the **Resources** folder.
 
-    ![added-resource-file](images/localization/added-resource-file.png)
+    ![Added resource file in .NET MAUI DataForm.](images/localization/added-resource-file.png)
 
 6. Add the Name or Value pair in the Resource Designer of the `DataFormLocalization.fr.resx` file and change its corresponding value to the corresponding culture.
  
-    ![name-value-pair](images/localization/name-value-pair.png)
+    ![Name value pair in .NET MAUI DataForm.](images/localization/name-value-pair.png)
```
Comment (karthikraja-arumugam) [inline MAUI/DataForm/localization.md:62]: Localization name value pair in .NET MAUI DataForm.

File: MAUI/DataForm/layout.md (hunk 1 above)
Comment (Kerubananthan3761) [reply MAUI/DataForm/layout.md]: renamed.

File: MAUI/DataForm/layout.md (hunk 2 above)
Comment (Kerubananthan3761) [reply MAUI/DataForm/layout.md]: renamed.

File: MAUI/DataForm/localization.md (hunk 3 above)
Comment (Kerubananthan3761) [reply MAUI/DataForm/localization.md]: renamed.

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**